### ISSUE TITLE
simplexml won't work

### ISSUE THREAD

**Issue by standa** [thecodingmachine/safe#46]
Unless I get the idea wrong, simplexml return types will not work.

```php
namespace Safe;
// must return \SimpleXMLElement instead of Safe\SimpleXMLElement (class does not exist)
function simplexml_load_string(...): \SimpleXMLElement { ... }
function simplexml_load_file(...): \SimpleXMLElement { ... }
function simplexml_import_dom(...): \SimpleXMLElement { ... }
```

I believe this is a general problem of https://github.com/thecodingmachine/safe/blob/master/generator/src/WritePhpFunction.php#L51

**Comment by moufmouf** [comment]
Jesus!!! Let's fix this immediately!

**Comment by moufmouf** [comment]
Thanks a lot for the feedback. Fixed in #47.

**Comment by standa** [comment]
@moufmouf wow respect for resolving this so quickly! One more proof for us to continue using this great library!
